src/components/Species/index.tsx

```diff
@@ -109,9 +107,13 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
-type SpeciesSearchResultRow = Omit<Species, 'growthForm' | 'seedStorageBehavior' | 'ecosystemTypes'> & {
-  conservationStatus?: string;
+type SpeciesSearchResultRow = Omit<
+  Species,
+  'growthForm' | 'seedStorageBehavior' | 'ecosystemTypes' | 'conservationCategory' | 'rare'
+> & {
+  conservationCategory?: string;
```

Comment:
I don't think we need to omit conservationCategory and add it here as a string, because it's already a string

Reply:
Oh, no, I forgot typescript would complain about that, just leave it as it is

Reply:
Last question, do we want to show on the species table the category code? Or should we show the full label?

Reply:
I asked the same, and the answer was that we should only show the code: https://terraformation.atlassian.net/browse/SW-3127?focusedCommentId=15190